script/install-as-user

```diff
@@ -127,6 +127,9 @@ sed -r \
     -e "s,# constraint_disabling: false,  constraint_disabling: false," \
     config/database.yml-example > config/database.yml
 
+# create the template_utf8 template we'll use for our databases
+createdb -T template0 -E UTF-8 template_utf8
```

Comment:
Ah, I wondered about that, but it seemed to work fine for me without doing it, so I left it as-is just to make it simpler. Do you remember where it was an issue if you left it out?